spec/requests/authors_spec.rb

```diff
@@ -23,4 +23,25 @@
       end
     end
   end
+
+  describe 'POST #create' do
+    context 'when the author is created' do
```

Comment:
O que acontece quando um autor não for criado? Faltou criar um contexto pra essa situação